lib/api/sitemap/index.ts

```diff
@@ -78,6 +78,39 @@ export const getSiteMapNewsData = async (locale: string) => {
   return { siteTitle, news: newsEntries };
 };
 
+export const getSitemapGalleryData = async (locale: string) => {
+  const site = getSiteFromLocale(locale);
+
+  const query = gql`
+    query ImageSitemapData($site: [String]) {
+      galleries: galleriesEntries(site: $site) {
+        ... on galleries_gallery_Entry {
+          uri
+          assetAlbum {
```

Comment:
issue: Canto albums can contain up to 6 different asset types: image, video, document, audio, presentation, and other. 

Add a filter here so that we only query supported schemes. Check `lib/api/galleries/schema.ts` for the `SupportedCantoSchemes` schema and add them with a `whereIn` argument like: `whereIn: {key: "scheme", values: ["image","video","document"]}`

check out `getGalleryData` in `lib/api/galleries/index.ts` for an example of how the bindings are set up